source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACOResourceResolvers.h

```diff
@@ -0,0 +1,18 @@
+//
+//  ACOResourceResolvers.h
+//  ACOResourceResolvers
+//
+//  Copyright © 2018 Microsoft. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import <UIKit/UIKit.h>
+#import "ACOIResourceResolver.h"
+
+@interface ACOResourceResolvers:NSObject
+
+- (instancetype)init;
+- (NSObject<ACOIResourceResolver> *)getResourceResolverForScheme:(NSString *)scheme;
```

Comment:
>NSObject [](start = 3, length = 8)

Prefer `id` over `NSObject`. It doesn't make a big difference, but it's more idiomatic and slightly more general. There are sometimes objects that don't derive from NSObject.

Reply:
thanks!

---
In reply to: [202425402](https://github.com/Microsoft/AdaptiveCards/pull/1702#discussion_r202425402) [](ancestors = 202425402)

Reply:
tried removing it, but received compilation error.

---
In reply to: [202456632](https://github.com/Microsoft/AdaptiveCards/pull/1702#discussion_r202456632) [](ancestors = 202456632,202425402)